pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -227,7 +227,7 @@ stdenv.mkDerivation (finalAttrs: {
     "-L" "${icu.out}/lib"
     "-I" "${icu.dev}/include"
     "-pch"
-  ] ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
+  ] ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
     "-device ${qtPlatform stdenv.hostPlatform}"
     "-device-option CROSS_COMPILE=${stdenv.cc.targetPrefix}"
   ]
```

Comment:
IMO this looks like a case where some variation of `stdenv.hostPlatform != stdenv.buildPlatform` is correct

Reply:
6d76aa30011ff92450ba5ab7397cdf98fdbc03cd